scripts/extract_dts_includes.py

```diff
@@ -397,14 +400,21 @@ def extract_single(node_address, yaml, prop, key, prefix, defs, def_label):
 
   return
 
-def extract_property(yaml, node_address, y_key, y_val, names, prefix, defs):
+def extract_property(node_compat, yaml, node_address, y_key, y_val, names, prefix, defs, label_override):
 
   node = reduced[node_address]
-  def_label = convert_string_to_label(get_compat(node)).upper()
-  def_label += '_' + node_address.split('@')[-1].upper()
+
+  if yaml[node_compat].get('base_label') is not None:
```

Comment:
Would `if 'base_label' in yaml[node_compat]` work? Might be a bit more idiomatic.